lib/index.js

```diff
@@ -98,8 +98,16 @@ export default {
           return null;
         }
 
+        const removePHP = str => str.replace(/<\?(?:php)?(?:[\s\S])+?\?>/gi, (match) => {
+          const newlines = match.match(/\r?\n|\r/g);
+          const newlineCount = newlines ? newlines.length : 0;
```

Comment:
This has to be done this way, because if no matches are found, `match` returns undefined. `undefined.length` throws an error, so I'm getting the matches, then getting the count based on whether it returned anything or not.